### ISSUE TITLE
Order by cannot support multiple clustering columns with different order

### ISSUE THREAD

**Issue by jindai1783** [masumsoft/express-cassandra#106]
We would like to support queries like that:
```
SELECT * FROM users 
  WHERE user_id = <uuid> AND status = 'verified' AND timestamp > <timestamp_value> 
  ORDER BY status ASC, timestamp DESC 
  LIMIT 25;
```
I can see in the docs that we can use:
```
$orderby: { $asc: 'status' }
```
or 
```
$orderby: { $asc: ['status', 'timestamp'] }
```
but in our case we need something like that but it seems that it is not supported yet:
```
where['$orderby'] = {
     '$asc': 'status',
     '$desc': 'timestamp'
};
```
Is this the case?

We can send a pull request for this.

**Comment by masumsoft** [comment]
I cannot exactly recall why this case was intentionally blocked in the code. There is a specific error handling for multiple orderby column queries that checks if the query object has multiple orderby  and throws error. I guess multiple order by was not supported by cassandra back then. Anyways, a pull request would be awesome!

**Comment by jindai1783** [comment]
Yes, the possible scenario is that it was not supported in the past.

We just sent a pull request that is fixing this specific check about multiple orderby :)

**Comment by masumsoft** [comment]
Fixed in v1.8.3
